apps/reward-manager/src/notifier.ts

```diff
@@ -0,0 +1,48 @@
+import DDClient, { DDEvent } from "./dd_client";
```

Comment:
Can we remove this in favor of the existing data dog logging package in the monorepo?